tests/time_tests/test_runner/.automation/desktop_test_config_cache.yml

```diff
@@ -0,0 +1,543 @@
+- device:
```

Comment:
Have you forgotten to update desktop_test_config.yml  for use_gpu_cache=true?

Reply:
No, there is separate `desktop_test_config_cache.yml` for model cache preparation. It should be used together with `timetest_infer_cache`